corehq/apps/aggregate_ucrs/tests/test_aggregation.py

```diff
@@ -249,7 +249,7 @@ def test_aggregate_table(self):
         self.assertEqual(SmallInteger, type(table.columns['pregnant_in_month'].type))
 
         # basic check on secondary column
-        self.assertEqual(Integer, type(table.columns['fu_forms_in_month'].type))
+        self.assertEqual(SmallInteger, type(table.columns['fu_forms_in_month'].type))
```

Comment:
how did this change from integer to small integer? not seeing it in this PR

As a side note, I've been very hesitant to make aggregate data use small integers because 32k of something can be very easy to hit on ICDS or other large projects. Seems like a fine default as long as it's documented to be changeable somewhere

Reply:
it's because `sum` now [defaults to using the type of its referenced column](https://github.com/dimagi/commcare-hq/pull/21639/files#diff-618ed31918186c40875fc3fd391adea9R330), which for `count` [is a small_integer](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/app_manager/helpers.py#L90).

Good to know re: not wanting to use it. It can definitely be overridden by changing the type of the source, though not the most obvious behavior. I've been mulling over how best to do datatypes here anyway, might end up changing it.

Reply:
Thanks that makes sense. 

It'll likely make sense to be able to override that later, but I imagine that that should be a project-specific decision